x/lockup/client/rest/rest.go

```diff
@@ -0,0 +1,26 @@
+package rest
```

Comment:
Do we not need to add these?

Reply:
Added legacy rest endpoints

Reply:
But it won't be that necessary as it's legacy

Reply:
Oh I see